src/data/data.js

```diff
@@ -15,14 +15,19 @@ const pageData = {
           href: 'Register'
         }
       },
+      {
+        component: 'page-title',
```

Comment:
Please camelCase format to name the components.

Reply:
Done!

Reply:
It still requires to change to camelCase.
<img width="423" alt="Screen Shot 2020-08-07 at 7 47 19 PM" src="https://user-images.githubusercontent.com/36742076/89698442-dff9f700-d8e6-11ea-92a3-1716d37ece94.png">